x-pack/plugins/endpoint/public/embeddables/resolver/store/camera/methods.ts

```diff
@@ -17,6 +18,16 @@ export function animatePanning(
   targetTranslation: Vector2,
   duration: number
 ): CameraState {
+  const initialTranslation = translation(state)(startTime);
+  const translationDistance = distance(targetTranslation, initialTranslation);
+
+  if (translationDistance === 0) {
+    return {
+      ...state,
+      panning: undefined,
```

Comment:
❔ Should `animation` also set to `undefined` here?

Reply:
I'm looking specifically at https://github.com/elastic/kibana/blob/4d80b07cf8376a36100a595234f13045d5bc6ebb/x-pack/plugins/endpoint/public/embeddables/resolver/store/camera/selectors.ts#L77 and it looks like `if animation !== undefined` it takes `panningDistance` as 0 there and then feeds it into a div/0 below when it tries to get the `adjustedScale`

Or am I maybe missing something there?

Reply:
Funny enough we didn't see the issue with `scale` specifically, but with the projectionMatrix below and how the clipping planes were being calculated, leading to the screenToNDC value returning NaN and the current bug: 

https://github.com/elastic/kibana/blob/4d80b07cf8376a36100a595234f13045d5bc6ebb/x-pack/plugins/endpoint/public/embeddables/resolver/store/camera/selectors.ts#L435

It didn't come up when I paired with @oatkiller on this one, but I expect that it's defaulted to undefined before panning takes place.  With that said, the intent will come across better too if I explicitly set it to undefined there. Will update. Thanks!